app/layout.tsx

```diff
@@ -1,7 +1,7 @@
 //
-// This source file is part of the Stanford Biodesign Digital Health Next.js Template open-source project
```

Comment:
As this repository is most likely to only contain packages and doesn't actually have a frontend-app, we may as well remove the entire `app` directory, right?